buku

```diff
@@ -4962,23 +4962,23 @@ def print_single_rec(row: BookmarkVar, idx: int=0, columns: int=0):  # NOQA
             return
 
         INDENT = 5
-        ln_num = 1
         fillwidth = columns - INDENT
+        desc_lines = [s for line in row.desc.splitlines()
+                        for s in textwrap.wrap(line, width=fillwidth) or ['']]
+        TR = str.maketrans(',-', '-,')  # we want breaks after commas rather than hyphens
+        tag_lines = textwrap.wrap(row.tags.translate(TR), width=fillwidth)
```

Comment:
`wrap()` inserts linebreaks after hyphens by default; which is not really a good thing when displaying tags. On the other hand, it doesn't insert linebreaks after commas (unless those are followed by whitespace); so what we want is to replace `-` with `,` and vice versa before passing tags string to `wrap()` (then revert the change during the printing).